packages/processors/src/processors/registry/handlers/profileMetadataUpdated.handler.ts

```diff
@@ -0,0 +1,65 @@
+import { Changeset, ProjectType } from "@grants-stack-indexer/repository";
+import { ChainId, ProcessorEvent } from "@grants-stack-indexer/shared";
+
+import { IEventHandler, ProcessorDependencies } from "../../../internal.js";
+import { ProjectMetadata, ProjectMetadataSchema } from "../../../schemas/index.js";
+
+type Dependencies = Pick<
+    ProcessorDependencies,
+    "projectRepository" | "evmProvider" | "metadataProvider" | "logger"
+>;
+/**
+ * Handles the ProfileMetadataUpdated event for the Registry contract from Allo protocol.
+ */
+export class ProfileMetadataUpdatedHandler
+    implements IEventHandler<"Registry", "ProfileMetadataUpdated">
+{
+    constructor(
+        readonly event: ProcessorEvent<"Registry", "ProfileMetadataUpdated">,
+        readonly chainId: ChainId,
+        private dependencies: Dependencies,
+    ) {}
+    async handle(): Promise<Changeset[]> {
```

Comment:
```suggestion
    ) {}
    /* @inheritdoc */
    async handle(): Promise<Changeset[]> {
```
or a more specific description